tracer/src/Datadog.Trace/Propagators/B3MultipleHeaderContextPropagator.cs

```diff
@@ -67,7 +67,7 @@ private B3MultipleHeaderContextPropagator()
 
             TelemetryFactory.Metrics.RecordCountContextHeaderStyleExtracted(MetricTags.ContextHeaderStyle.B3Multi);
             var samplingPriority = ParseUtility.ParseInt32(carrier, carrierGetter, Sampled);
-            spanContext = new SpanContext(traceId, parentId, samplingPriority, serviceName: null, null, rawTraceId, rawSpanId);
+            spanContext = new SpanContext(traceId, parentId, samplingPriority, serviceName: null, null, rawTraceId, rawSpanId, isRemote: true);
```

Comment:
I think this change is from another branch which is already merged into `master`. You may want to rebase this branch on `master` to see the updated diff.